boards/arm/qemu_cortex_m0/nrf_timer_timer.c

```diff
@@ -255,3 +255,8 @@ uint32_t sys_clock_cycle_get_32(void)
 	k_spin_unlock(&lock, key);
 	return ret;
 }
+
+uint64_t sys_clock_cycle_get_64(void)
+{
+	return 0;
```

Comment:
@ioannisg - would be great to get your input here. It seems there is no 64-bit timer register on this platform.

For software emulation, I would like to increment a 32-bit "top half" each time a 32-bit cycle counter overflows (assuming a 1:1 ratio), shift and then OR it with the regular 32-bit value.

In this driver, where is the best location to increment the 32-bit "top half"?